ray-operator/controllers/ray/common/ingress.go

```diff
@@ -87,7 +86,7 @@ func BuildIngressForHeadService(cluster rayv1.RayCluster) (*networkingv1.Ingress
 	// Get ingress class name from rayCluster annotations. this is a required field to use ingress.
 	ingressClassName, ok := cluster.Annotations[IngressClassAnnotationKey]
 	if !ok {
-		logrus.Warn(fmt.Sprintf("ingress class annotation is not set for cluster %s/%s", cluster.Namespace, cluster.Name))
+		log.Info(fmt.Sprintf("ingress class annotation is not set for cluster %s/%s", cluster.Namespace, cluster.Name))
```

Comment:
Note, there is the logger provided by controller-runtime that can be retrieved from the context, e.g.:

```go
log := ctrl.LoggerFrom(ctx)
```

This contains the reconciliation request id, and can be customised with common key / value pairs (e.g. namespace and name) with the `LogConstructor` option of the controller builder.

But a global variable can be a good approximation at the moment.